Mathlib/MeasureTheory/Function/L2Space.lean

```diff
@@ -184,15 +184,15 @@ theorem integrable_inner (f g : α →₂[μ] E) : Integrable (fun x : α => ⟪
     (AEEqFun.integrable_iff_mem_L1.mpr (mem_L1_inner f g))
 
 private theorem add_left' (f f' g : α →₂[μ] E) : ⟪f + f', g⟫ = inner f g + inner f' g := by
-  simp_rw [inner_def, ← integral_add (integrable_inner f g) (integrable_inner f' g), ←
-    inner_add_left]
+  simp_rw [inner_def]
+  rw [← integral_add (integrable_inner f g) (integrable_inner f' g)]
```

Comment:
I'm not quite sure why this change is necessary. (Making everything one `simp_rw` certainly fails now.)
A careful look if there are deeper issues would be appreciated.

Reply:
Never mind, that was elsewhere. 
There is nothing in the conclusion of `integrable_inner` that identifies `𝕜`. 
It probably should be explicit, and I assume that before it could be synthesized using simple unification of instance arguments, which now fails.

Reply:
Thanks for the clarification! I agree with your fixes.
I'll put "make k in `integrable_inner` explicit" on my follow-up list of PR - in the new year.